blazor/ui-kit/build-your-first-blazor-app-with-blocks.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: Build your first Blazor app with our blocks | Syncfusion
+description: Learn all about building your first Blazor app using our blocks from the Essential Blazor UI Kit in Syncfusion Essential Studio and more.
+platform: Blazor
+control: UI Kit
+documentation: ug
+---
+
+# Build your first Blazor Web App with our blocks
+
+## Create a new Blazor Web App
+A Blazor Web App is used for this example. To create a new app, follow the official setup guide [here](https://learn.microsoft.com/en-us/training/modules/build-your-first-blazor-web-app/3-exercise-configure-environment?pivots=vscode). This tutorial then walks through the step-by-step process of adding a simple sign-in block to the newly created app, named MyBlazorApp.
```

Comment:
Both # and ## titles are contradictory with first line - "A Blazor Web App is used for this example.".